### PR TITLE
Assassin set and Mana Pearl item db and function

### PR DESCRIPTION
**Pull request by HoraK-FDF**
- tweaked assassins set and added assassin mask, assassin ring, assassin amulet and the mana-pearl
- changed price of enchanters to be like other trinklets
- aligned trinklets db file

### CODE REVIEW

File: world/map/db/item_db_head.txt [hunk 1]
```diff
@@ -226,3 +226,4 @@
 5264,   BlueWolfHelmet,                 5,      1000,       500,        50,         0,      3,      0,      0,      0,      2,      256,        0,      0,      0,      {},                                                                     {}
 5266,   CloverHat,                      5,      1000,       500,        50,         0,      3,      0,      0,      0,      2,      256,        0,      0,      0,      {},                                                                     {bonus bLuk, 1;}
 //ID,   Name___________________,        Type,   Price,      Sell,       Weight,     ATK,    DEF,    Range,  Mbonus, Slot,   Gender, Loc,        wLV,    eLV,    View,   {UseScript},                                                            {EquipScript}
+5267,   AssassinMask,                   5,      30000,      3000,       10,         0,      16,     0,      -5,     0,      2,      256,        0,      0,      0,      {},                                                                     {callfunc "BrawlingItem"; bonus bMdef, 5; bonus bAgi, 9; bonus bDex, 1; bonus bDoubleAddRate, 15; bonus bCritical, 15; bonus bAspdRate, 15;}
```
Comment (pazkero) [inline world/map/db/item_db_head.txt:229]: 16 DEF is too much for the mask.
0 DEF and 16 EVD is better.

File: world/map/db/item_db_trinket.txt [hunk 2]
```diff
@@ -19,5 +19,8 @@
...
+5268,   AssassinRing,                   5,      0,          0,          1,          0,      0,      0,      0,      0,      2,      128,        0,      0,      0,      {},                                                                     {callfunc "BrawlingItem"; bonus bAgi, 1; bonus bCritical, 5; bonus bAspdRate, 5;}
+5269,   AssassinAmulet,                 5,      0,          0,          1,          0,      0,      0,      0,      0,      2,      16,         0,      0,      0,      {},                                                                     {callfunc "BrawlingItem"; bonus bAgi, 1; bonus bDoubleAddRate, 5; bonus bHit, 25;}
+5270,   ManaPearl,                      5,      0,          0,          0,          0,      0,      0,      11,     0,      2,      16,         0,      0,      0,      {},                                                                     {callfunc "ManaPearlItem"; bonus bInt, 1; bonus bMdef, 11; bonus bMaxSP, 111;}
```
Comment (pazkero) [inline world/map/db/item_db_trinket.txt:26]: 111 MP is a lot; That's over half of what I currently have.

File: world/map/npc/items/manapearl_item.txt [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    set @manapearl_mexp_needed, 5404;
+
+    if ( @mexp >= @manapearl_mexp_needed )
+        goto L_Return;
+
+    message strcharinfo(0), "You are not expirienced enough to keep up the mana flow that is needed to stay in resonance with the Mana Pearl.";
+
+    // This is for debug
+    if (getgmlevel() >= 60)
```
Comment (pazkero) [inline world/map/npc/items/manapearl_item.txt:21]: Better use `if (debug)` then, or remove, but it is weird to allow GMs to equip the ring and get -7 agi.

Looks a lot like copy-paste

**Review by pazkero — CHANGES_REQUESTED** [review]
Please fix those and decide for style (HP or EVD, but NOT defense)

File: world/map/db/item_db_trinket.txt (hunk 2 above)
Comment (HoraK-FDF) [reply world/map/db/item_db_trinket.txt]: 55 MP now

File: world/map/db/item_db_head.txt (hunk 1 above)
Comment (HoraK-FDF) [reply world/map/db/item_db_head.txt]: reduced def to 6 (somehow I could not bear the thought of a 0 def item) and as you said 1 def = 7 hp i turned the remaining 10 def into 70 hp. i hope that's works for you.

**Review by pazkero — APPROVED** [review]
OK

**Pull request merged**